.github/workflows/build-and-release.yml

```diff
@@ -0,0 +1,45 @@
+name: Build and release
+
+on:
+  push:
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+          registry-url: https://registry.npmjs.org/
+      - run: npm ci
+      - run: npm run test:ci
+        env:
+          CI: true
+
+  release:
+    name: Release
+    if: ${{ github.ref == 'refs/heads/master' }}
+    runs-on: ubuntu-latest
+    needs: [build]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14
+          registry-url: https://registry.npmjs.org/
+      - run: npm ci
+      - name: Release
+        run: npx semantic-release --debug
+        env:
+          CI: true
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
Hey, I just found [this](https://github.com/actions/setup-node/blob/main/action.yml#L16):

```
registry-url - Will set the registry in a project level .npmrc and .yarnrc file, 
               and set up auth to read in from env.NODE_AUTH_TOKEN
```

Should we change `NPM_TOKEN` to `NODE_AUTH_TOKEN` ?

Reply:
My understanding was that `semantic-release` was taking care of everything using `NPM_TOKEN`, and it did when I tried locally.
```
[semantic-release] [@semantic-release/npm] › ℹ  Verify authentication for registry https://registry.npmjs.org/
[semantic-release] [@semantic-release/npm] › ℹ  Wrote NPM_TOKEN to /private/var/folders/***/.npmrc
protractor-cucumber-framework-publisher
[semantic-release] › ✔  Completed step "verifyConditions" of plugin "@semantic-release/npm"
```
But at this point we can try to set both and see how it behaves.